src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -108,22 +124,36 @@ export default defineComponent<Props>({
         },
         currency: {
             type: String as PropType<Currency>,
-            default: CURRENCY.USD,
+            default: undefined,
         },
     },
-    setup(props, { emit }:SetupContext) {
+    setup(props) {
+        const { i18nDayjs } = useI18nDayjs();
+        const setBasicDateFormat = (date) => (date ? dayjs(date).format('YYYY-MM-DD') : undefined);
         const state = reactive({
-            proxySelectedDates: useProxyValue('selectedDates', props, emit),
-            proxyCurrency: useProxyValue('currency', props, emit),
             isFull: computed<boolean>(() => props.size === WIDGET_SIZE.full),
+            fullDataTag: computed<string>(() => (props.widgetLink ? 'a' : 'router-link')),
+            dateLabel: computed<TranslateResult|undefined>(() => {
+                const start = setBasicDateFormat(props.dateRange?.start);
+                const end = setBasicDateFormat(props.dateRange?.end);
+                if (start && end) {
+                    return `${start} ~ ${end}`;
+                }
+                if (start && !end) {
+                    const today = dayjs().utc();
+                    const diff = today.diff(dayjs(start), 'day', true);
+                    if (diff < 1) return i18n.t('Today'); // song-lang
+                    if (diff >= 6 && diff < 7) return i18n.t('Past 7 days'); // song-lang
+                    return i18nDayjs.value(start).from(today.subtract(1, 'day'));
+                }
+                return undefined;
+            }),
```

Comment:
suggestion: Timezone conversion to utc is not required.

WidgetFrame component just display times by user timezone.
So, I think dateLabel state must be user timezone.